doc/build_instructions/windows_msvc.md

```diff
@@ -28,13 +28,17 @@ __NOTE:__ You need to manually make sure and doublecheck if the system you are b
    - With the "Download debug binaries (...)" option enabled.
    - If in doubt, run the installer again and choose "Modify".
    - You are going to need the 64-bit version of python if you are planning to build the 64-bit version of openage, and vice versa.
+ 
+    __Note:__ You need to install Python 3.7.x currently, because 3.8 deals with finding DLLs differently (see #1201) which would result in errors down the line
+ 
  - [CMake](https://cmake.org/download/)
 
 ### Python Modules
  Open a command prompt at `<Python 3 installation directory>/Scripts`
 
     pip install cython numpy pillow pygments pyreadline Jinja2
 
+_Note:_ Make sure the Python 3 instance where you're installing these scripts is the one you call `python` in CMD
```

Comment:
`python` or `python3` should be sufficient. python could be python 2.7 for some as well. They  would need to test that actually. Makes sense to state that as well I guess to test if `python` points to Python 2 or 3

Reply:
also a line in between this and the next headline for people reading the raw text would be good